vault/logical_system.go

```diff
@@ -1406,6 +1456,210 @@ func (b *SystemBackend) handleRotate(
 	return nil, nil
 }
 
+func (b *SystemBackend) handleWrappingWrap(
+	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	if req.WrapTTL == 0 {
+		return logical.ErrorResponse("endpoint requires response wrapping to be used"), logical.ErrInvalidRequest
+	}
+
```

Comment:
So, this is a good question. Nothing currently prevents a backend from having an empty `Data` wrapped. Sometimes the fact that there is no data is itself significant. So I think we want to allow this unless there is a reason not to.